dali/pipeline/operators/decoder/host_decoder_test.h

```diff
@@ -39,7 +39,7 @@ class HostDecodeTestBase : public GenericDecoderTest<ImgType> {
   }
 
   inline uint32_t GetImageLoadingFlags() const override {
-    return image_type_;
+    return t_loadJPEGs | t_loadPNGs | t_loadTiffs;
```

Comment:
Can't we just get rid of this function?

Reply:
we can get rid of so many things :), but I don't want to waste more time in grooming those tests now